pkg/ibmvpc/explainability_test.go

```diff
@@ -740,3 +740,32 @@ func TestInputValidityMultipleVPCContext(t *testing.T) {
 		"- test-vpc0-ky, test-vpc1-ky - please add the name of the config to the src/dst name",
 		err11.Error())
 }
+
+func TestMultiExplainability(t *testing.T) {
+	vpcsConfig := getConfig(t, "tgw_larger_example")
+	require.NotNil(t, vpcsConfig, "vpcsConfig equals nil")
+	groupedConns := make(map[string]*vpcmodel.GroupConnLines)
+	nodesConn := make(map[string]*vpcmodel.VPCConnectivity)
+	for i := range vpcsConfig {
+		thisConn, err := vpcsConfig[i].GetVPCNetworkConnectivity(false)
+		if err != nil {
+			fmt.Printf("%v. %s", i, err.Error())
+		}
+		require.Nil(t, err)
+		groupedConns[i] = thisConn.GroupedConnectivity
+		nodesConn[i] = thisConn
+	}
+	inputMultiExplain := vpcmodel.CreateMultiExplanationsInput(vpcsConfig, groupedConns)
+	multiExplain := vpcmodel.MultiExplain(inputMultiExplain, nodesConn)
+	// sanity check: number of entries did not change (187) and no errors
+	i := 0
+	for _, explain := range multiExplain {
+		if explain.Error() != "" {
+			// todo: issues with 172.128.0.0/9 https://github.com/np-guard/vpc-network-config-analyzer/issues/535
+			//       after the issue is solved add sanity that no error
+			fmt.Printf("%v. %v\n", i, explain.Error())
+		}
+		i++
+	}
+	require.Equal(t, i, 187)
```

Comment:
better not use hard-coded numbers

Reply:
In testing we compare to hard code text all the time. So I don't see an issue with an hard coded number.
Regardless of the hard-coded number, I'm not sure this specific comparison is very valuable. Perhaps I will just verify there is no error and that the number of explanation structs equals the number of inputs.

Reply:
Agreed, only I'm not sure in this specific case changing the number to constant will make any difference.
Anyways, I've changed the test so this discussion no longer relevant.

Reply:
I've added a test on the actual output string.